_raml/apps/products/examples/patch_product_productid_request.json

```diff
@@ -35,4 +35,64 @@
   "op": "replace",
   "path": "/producttype",
   "value": "6489A241-7B85-54B0-0B69-C0A8020205ED"
+},
+{
+    "op": "add",
+    "path": "/isNew",
+    "value": true
+},
+{
+    "op": "add",
+    "path": "/newnessDate",
+    "value": "2023-08-08T22:00:00Z"
+},
+{
+    "op": "add",
+    "path": "/video",
+    "value": { "type":"YouTube", "id":"XXXXXXXXXXX" }
+},
+{
+    "op": "add",
+    "path": "/isManufacturerPriceRRP",
+    "value": false
+},
+{
+    "op": "add",
+    "path": "/referencePriceInfo",
+    "value": {"refAmount":1, "refUnit":"piece", "refContentAmount":2}
+},
+{
+    "op": "add",
+    "path": "/isShippingLinkVisible",
+    "value": true
+},
+{
+    "op": "add",
+    "path": "/customisationInfo",
+    "value": {"isCustomizable":1, "customizableTextLength":50, "customizableHeadlineText":"headline text"}
+},
+{
+    "op": "add",
+    "path": "/title",
+    "value": "My title"
+},
+{
+    "op": "add",
+    "path": "/productDataSheet",
+    "value": {"fileName":"myfilename.jpg", "url":"https://www.example.com/file.jpg"}
+},
+{
+    "op": "add",
+    "path": "/shippingMethodsRestrictedTo",
+    "value": ["64BDD860-786B-D5AB-F659-AC1200030B22"]
+},
+{
+    "op": "add",
+    "path": "/isVATInfoVisible",
```

Comment:
If we add this operation here, doesn't at least this path need to be added to the products.raml?

Reply:
Right you are :D